tensorflow/tools/api/golden/v2/tensorflow.summary.pbtxt

```diff
@@ -10,7 +10,7 @@ tf_module {
   }
   member_method {
     name: "audio"
-    argspec: "args=[\'name\', \'data\', \'sample_rate\', \'step\', \'max_outputs\', \'encoding\', \'description\'], varargs=None, keywords=None, defaults=[\'None\', \'3\', \'None\', \'None\'], "
+    argspec: "args=[\'name\', \'data\', \'sample_rate\', \'step\', \'max_outputs\', \'encoding\', \'description\'], varargs=None, keywords=None, defaults=[\'3\', \'None\', \'None\'], "
```

Comment:
Can you revert the changes in this file?

Reply:
+1, these changes are also blocking. I think you may need to merge from master, then re-run the update golden script